Dockerfile

```diff
@@ -13,7 +13,22 @@ RUN go mod download
 # Copy the source code from the current directory to the Working Directory inside the container
 COPY . .
 
-RUN go test ./... \
-  && go build -ldflags='-s -w' -o main .
+RUN go build -ldflags='-s -w' -o main .
 
-ENTRYPOINT ["/app/main"]
+# End from the latest alpine image
+FROM alpine:latest
+
+# add bash and timezone data
+RUN \
```

Comment:
Add `# hadolint ignore=DL3008` above this line to ignore another linter warning.